teams/teams-ps/teams/Set-CsTeamsVoiceApplicationsPolicy.md

```diff
@@ -8,27 +8,36 @@ schema: 2.0.0
 # Set-CsTeamsVoiceApplicationsPolicy
 
 ## SYNOPSIS
+
 Modifies an existing Teams voice applications policy.
 
 ## SYNTAX
 
-```
+```ps
 Set-CsTeamsVoiceApplicationsPolicy [-AllowAutoAttendantAfterHoursGreetingChange <Boolean>]
  [-AllowAutoAttendantBusinessHoursGreetingChange <Boolean>]
- [-AllowAutoAttendantHolidayGreetingChange <Boolean>] [-AllowCallQueueMusicOnHoldChange <Boolean>]
+ [-AllowAutoAttendantHolidayGreetingChange <Boolean>] [-AllowAutoAttendantBusinessHoursChange <Boolean>] [-AllowAutoAttendantTimeZoneChange <Boolean>] [-AllowAutoAttendantLanguageChange <Boolean>] [-AllowAutoAttendantHolidaysChange <Boolean>] [-AllowAutoAttendantBusinessHoursRoutingChange <Boolean>] [-AllowAutoAttendantAfterHoursRoutingChange <Boolean>]
+ [-AllowAutoAttendantHolidayRoutingChange <Boolean>] [-AllowCallQueueMusicOnHoldChange <Boolean>]
  [-AllowCallQueueOverflowSharedVoicemailGreetingChange <Boolean>]
  [-AllowCallQueueTimeoutSharedVoicemailGreetingChange <Boolean>]
- [-AllowCallQueueWelcomeGreetingChange <Boolean>] [[-Identity] <String>]
+ [-AllowCallQueueWelcomeGreetingChange <Boolean>] [-AllowCallQueueOptOutChange <Boolean>] [-AllowCallQueueAgentOptChange <Boolean>] [-AllowCallQueueMembershipChange <Boolean>] [-AllowCallQueueRoutingMethodChange <Boolean>] [-AllowCallQueuePresenceBasedRoutingChange <Boolean>]
+ [-CallQueueAgentMonitorMode <Disabled | Monitor | Whisper | Barge | Takeover>]
+ [-CallQueueAgentMonitorNotificationMode <Disabled | Agent>]
+ [-AllowCallQueueLanguageChange <Boolean>] [-AllowCallQueueOverflowRoutingChange <Boolean>]
+ [-AllowCallQueueTimeoutRoutingChange <Boolean>] [-AllowCallQueueNoAgentsRoutingChange <Boolean>]
+ [-AllowCallQueueConferenceModeChange <Boolean>] [[-Identity] <String>]
  [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-TeamsVoiceApplicationsPolicy is used for Supervisor Delegated Administration which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations. 
+
+TeamsVoiceApplicationsPolicy is used for Supervisor Delegated Administration which allows tenant admins to permit certain users to make changes to auto attendant and call queue configurations.
 
 ## EXAMPLES
 
 ### EXAMPLE 1
-```
+
+```ps
```

Comment:
please whenever you intend to use powershell code block use powershell instead of ps